google-cloud-compute-v1/acceptance/google/cloud/compute/v1/test_use_compute_pagination.rb

```diff
@@ -0,0 +1,41 @@
+require 'securerandom'
+require "minitest/autorun"
+
+require "google/cloud/compute/v1/zones"
+require "google/cloud/compute/v1/regions"
+
+$default_zone = 'us-central1-a'
+$default_project = ENV['PROJECT_ID']
+
+class TestUseComputePagination < Minitest::Test
+
+  def setup
+    @client = Google::Cloud::Compute::V1::Zones::Rest::Client.new
+    if $default_project.eql? NIL
```

Comment:
Style: Please use the `nil` keyword rather than the `NIL` constant. (And in fact, better: `$default_project.nil?`)

Reply:
done